### ISSUE TITLE
Remove the Diagnostics "Old placeholder file is in the way" check

### ISSUE THREAD

**Issue by petecooper** [textpattern/textpattern#1656]
Could we consider removing the "Old placeholder file is in the way" check / warning from diagnostics in 4.9?:

![re](https://user-images.githubusercontent.com/226578/111667208-632fe380-880c-11eb-967e-2d4ad7bca43d.png)

It's a throwback to (veeeery) old Textpattern, and it looks like some CMSes residing in these subdirectories trigger this, which makes it a false positive in the current way of thinking. It might not be necessarily ideal to have nested CMSes this way, but it works, so I say 'live and let live'.

**Comment by phiw13** [comment]
Why ?

Imagine I have a non-Textpattern managed folder on my server named “books”.
Later on I create a section named “books”.

Conflict! Thank you Textpattern for warning me.

**Comment by petecooper** [comment]
- Fair. But by that workflow, you don't have "old placeholders" either. A web server can trivially be configured to try 'real' files and directories first, and fall back to e.g. index.php if they don't work. My point was perhaps more to say both can readily co-exist, and this red text might not be warranted.

**Comment by Bloke** [comment]
> A web server can trivially be configured to try 'real' files and directories first, and fall back to e.g. index.php if they don't work.

Isn't that standard? Not sure but I seem to recall that the last time this happened to me I couldn't figure out why the code in my contact section's page template wasn't running. Turns out the site had a /contact directory and the web server was using that in deference to index.php. Had I checked Diagnostics earlier, I might have noticed the warning and twigged what was going on.

So, yeah, this warning does have value for debugging. But I'm not sure calling it a 'placeholder' is the right terminology any more. Perhaps we should relocate this issue to the pophelp/textpack repos and get the wording updated?

**Comment by petecooper** [comment]
>Isn't that standard?

It varies between vendors and products. I first encountered this years ago when I wanted a section called `index`, and the web server prioritised `index.php` over the legit subdirectory, I can't find the forum thread right now, suffice to say I got cross with it all sufficiently enough to start learning about configuring web servers to change the priority order.

>Perhaps we should relocate this issue to the pophelp/textpack repos and get the wording updated?

Yep. I'll open an issue.
